package.json

```diff
@@ -9,10 +9,8 @@
     "test:ci": "jest",
     "heroku-postbuild": "npm run build",
     "lint": "eslint --fix --ext .js resources/assets",
-    "docs:prepare": "gitbook install",
     "precommit": "pretty-quick --staged",
     "format": "prettier --write \"{docs,resources}/**/*.{js,md,scss,graphql}\"",
```

Comment:
Oh, gotta kill the `,` at the end of this line.

Reply:
ah yep just caught that! builds are passing now.